examples/rbd/plugin-deploy.sh

```diff
@@ -13,3 +15,15 @@ objects=(csi-provisioner-rbac csi-nodeplugin-rbac csi-config-map csi-rbdplugin-p
 for obj in "${objects[@]}"; do
 	kubectl create -f "./$obj.yaml"
 done
+
+if [[ -z $kms_base ]]; then
+	kms_base="../../../examples/kms/vault"
+fi
+
+cd "$kms_base" || exit 1
+
+objects=(vault csi-vaulttokenreview-rbac kms-config)
+
+for obj in "${objects[@]}"; do
+	kubectl create -f "./$obj.yaml"
+done
```

Comment:
if you review your own PR in GitHub, you can see a warning here: "No newline at end of file"